pep-0458.txt

```diff
@@ -260,20 +261,20 @@ Integrating PyPI with TUF
 A software update system must complete two main tasks to integrate with TUF.
 First, it must add the framework to the client side of the update system. For
 example, TUF MAY be integrated with the pip package manager. Thus, new versions
-of pip going forward SHOULD use TUF by default to download and verify packages
+of pip going forward SHOULD use TUF by default to download and verify distributions
 from PyPI before installing them. However, there may be unforeseen issues that
-might prevent users from installing or updating packages, including pip itself,
+might prevent users from installing or updating distributions, including pip itself,
 via TUF. Therefore, pip SHOULD provide an option e.g.,
-`--unsafely-disable-package-verification`, in order to work around such issues
+`--unsafely-disable-distribution-verification`, in order to work around such issues
```

Comment:
I prefer we continue to refer to "package" in the option. The term "distribution" isn't used at all in Pip's `--help` output, but the term packages occurs 9 times.

Reply:
You have a point. But it seems weird if we almost completely ban the word "package" from this document.